trap/src/main/java/com/cursorinsight/trap/util/TrapTime.kt

```diff
@@ -1,32 +1,76 @@
 package com.cursorinsight.trap.util
+import android.os.SystemClock
 
 /**
  * Utility class to calculate the unified timestamps.
  */
 internal class TrapTime {
     companion object {
-        var bootTime: Long = System.currentTimeMillis()
+        /**
+         * Both elapsedRealtime and uptimeMillis are reported relative to the system boot
+         * and we have to report all timestamps in UNIX epoch we have to determine the system
```

Comment:
Multiple "we have to"

Reply:
No, I've removed all the not used imports

Reply:
I've updated the description